drivers/nvme/nvme.c

```diff
@@ -31,28 +31,35 @@
 #include "NvmExpress.h"
 #include "NvmCtrlLib.h"
 
-
 #define DEVICE_INDEX_DEFAULT  0
+#define MAX_PCI_BUS_NUM 8
 
 static struct supported_device {
 	u16 vid;
 	u16 did;
 } SUPPORTED_DEVICES[] = {
-	{ .vid = 0x8086, .did = NVME_PCI_DID },
```

Comment:
With your change, we will not support VID=0x8086, DID=NVME_PCI_DID any more, right?